ui/src/app/views/workspace/component.spec.ts

```diff
@@ -89,16 +102,21 @@ describe('WorkspaceComponent', () => {
         ClarityModule.forRoot()
       ],
       declarations: [
-        WorkspaceComponent
+        WorkspaceComponent,
+        WorkspaceShareComponent
       ],
       providers: [
         { provide: ClusterService, useValue: new ClusterServiceStub() },
         { provide: CohortsService, useValue: new CohortsServiceStub() },
         { provide: Http, useValue: new HttpStub() },
+        { provide: ProfileService, useValue: new ProfileServiceStub() },
         { provide: SignInService, useValue: SignInService },
         { provide: WorkspacesService, useValue: new WorkspacesServiceStub() },
-        { provide: ActivatedRoute, useValue: activatedRouteStub }
-      ] }).compileComponents().then(() => {
+        { provide: ActivatedRoute, useValue: activatedRouteStub },
+        { provide: ServerConfigService,
+          useValue: new ServerConfigServiceStub({
+            gsuiteDomain: 'fake-research-aou.org'
+          }) }      ] }).compileComponents().then(() => {
```

Comment:
nit: please clean up this pile of parens/brackets

```
  {
    provide: ServerConfigService,
    useValue: new ServerConfigServiceStub({
      gsuiteDomain: 'fake-research-aou.org' 
    })
  }
]}).compileComponents().then(
```

Reply:
done